### ISSUE TITLE
fund channel with feerate lower than 1000

### ISSUE THREAD

**Issue by LightningHelper** [ElementsProject/lightning#4661]
### Issue and Steps to Reproduce
As far as I understand the minimum fee-rate is 1sat/byte, right?
This means that a fund command like this:
`lightning-cli fundchannel 0d....d6 4000000 10`
should not pass, and will throw an error, such as, the minimum feerate is 1000, correct?

I tried running with a feerate of 10, and the transaction passed with 1sta/byte, not sure it is the correct behavior. 
I think it should return an error like, error, fee-rate below the minimum accepted...

Thanks

**Comment by vincenzopalazzo** [comment]
Mh this is a good assertion, but I think by specification there is a "minimum" value, but there is a theoretical minimum value that under this value the transaction should be stack on the mem pool forever (not really forever). However, in practice, you can ask the c-lightning to create a transaction with an amount that you will force by the command line.

I see the c-lightning command line as a lower-level method to talk with the daemon and I aspect that the client that uses c-lightning will make this check.

But, this is only my opinion, and maybe it is possible to make a check inside the lightning daemon, in fact, there are already some commands that make some check on the amount of satoshi put inside the operation.

**Comment by kristapsk** [comment]
Proper way would be to check `mempoolminfee` of `getmempoolinfo` of Bitcoin Core's RPC. In practice, if somebody configures `minrelaytxfee` (`mempoolminfee` defaults to this if local mempool is not full) lower than 1 sat/vB, transaction will not relay to other nodes and will be stuck in your node only (although in theory you could convince some miner / pool directly to include tx fee below 1 sat/vB into block).

**Comment by LightningHelper** [comment]
I agree, but it shouldn't just change the fee to 1set/byte on its own, right? This is what happened to me.

**Comment by sangbida** [comment]
This issue is being closed as part of issue cleanup.

**Reason**: No recent activity (last updated 1666 da

If you believe this issue should remain open, please comment and we can reopen it.
